install/dc-detect-version.sh

```diff
@@ -8,12 +8,10 @@ fi
 
 echo "${_group}Initializing Docker Compose ..."
 
-# Some environments still use `docker-compose` even for Docker Compose v2.
```

Comment:
Is this ever true? I think some environments allow `docker-compose` to be used even on Docker Compose v2, but the official syntax should be `docker compose` instead of `docker-compose`. At least specified in the docs:
https://docs.docker.com/compose/migrate/

Reply:
We found it to be true in the wild, hence the accommodation. Could dig in history and find the PR for more detail. It was a linux distro that munged the name, don't remember which.

Reply:
See https://docs.docker.com/compose/install/standalone/ 
> For example type `docker-compose up` when using Compose standalone, instead of `docker compose up`.